### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by EhtesHamXa**
I answered C# Q.43 three days ago forget to create a pull request. Now I also updated .NET questions and updated the Readme.md ..XD
 There is also Q.32 on .NET  unanswered, I added a comment there if someone is sure about that he/she can check the box.

### CODE REVIEW

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Hello, @EhtesHamXa! Thanks very great PR, can you please fix conflicts before we merge it, if you need help I am here.
image

File: README.md [hunk 1]
```diff
@@ -475,6 +475,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://logophilia.top/"><img src="https://avatars.githubusercontent.com/u/80007889?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Eduard Pech</b></sub></a><br /><a href="#design-SimStim" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=SimStim" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/jjeshwani"><img src="https://avatars.githubusercontent.com/u/53613241?v=4?s=100" width="100px;" alt=""/><br /><sub><b>jjeshwani</b></sub></a><br /><a href="#design-jjeshwani" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=jjeshwani" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/EhtesHamXa"><img src="https://avatars.githubusercontent.com/u/41139956?v=4?s=100" width="100px;" alt=""/><br /><sub><b>EhtesHamXa</b></sub></a><br /><a href="#design-EhtesHamXa" title="Answering Questions">💬</a><a href="#data-EhtesHamXa" title="Data">🔣</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=EhtesHamXa" title="Code">💻</a></td>
```
Comment (Ebazhanov) [inline README.md:478]: @EhtesHamXa please revert it back, because this changes can cause some issue. Usually adding new contributor is coming by generating additional PR e.g. https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pull/1253
so no worries you will be contributor soon, but first please revert these changes.

Comment (EhtesHamXa) [reply README.md]: @Ebazhanov  , I reverted the changes back. Thanks.

**Review by Ebazhanov — APPROVED** [review]
Thanks again @EhtesHamXa for all those answers and new questions, great job :star:!

**Comment by Ebazhanov** [comment]
@all-contributors please add @EhtesHamXa for design, code
